app/build.gradle

```diff
@@ -40,4 +40,6 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    testImplementation 'org.mockito:mockito-inline:3.7.7'
+    testImplementation 'org.mockito:mockito-junit-jupiter:3.7.7'
```

Comment:
where do you get this dependencies?
you just need
testImplementation "org.mockito:mockito-core:3.+"

Reply:
done